packages/wallets/src/components/Base/ModalStepWrapper/ModalStepWrapper.scss

```diff
@@ -62,6 +62,7 @@
         background: var(--general-main-2, #ffffff);
         border-bottom: 0.2rem solid #f2f3f4;
         z-index: 1;
+        background-color: #ffffff;
```

Comment:
```suggestion
        background-color: var(--general-main-2, #ffffff);
```

Reply:
@hasan-deriv is this needed? as the background color is already defined on line 62 above